acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -83,7 +82,11 @@ public final void commit(Transaction t) {
         }
     }
 
-    public abstract void doCommit(Transaction t) throws IOException;
+    /**
```

Comment:
And if an object was deleted?  Can a single commit not contain multiple operations?  Like an add and a change? It just seems strange that callers would care specifically "was this an add?", but nothing else.

If it does make sense to say "the commit was/was not an add", It would still probably be preferable to return an enum member.